### ISSUE TITLE
how can identify cursor in visual mode 

### ISSUE THREAD

**Issue by newyorkthink** [gdh1995/vimium-c#974]
sometime cursor is hhere
![image](https://github.com/gdh1995/vimium-c/assets/87791643/e810a967-9444-4308-89fa-6c4ccd61d274)

sometime cursor is here
![image](https://github.com/gdh1995/vimium-c/assets/87791643/79ec6ef9-1017-4301-a37f-98bb37077916)

and i don't see the cursor

**Comment by shynekomaid** [comment]
You can toggle caret mode in visual mode using `c`. After you can toggle caret and select using `c` and `v`. I recommend to use `yv` to quick open visual mode and select word using hinting. To set carent in any place of screen just use `yvc`, `yc` used for "Go to forward". Visual, line and caret mode so complex and easy to use.

**Comment by gdh1995** [comment]
I agree with @shynekomaid . `yv` (`LinkHints.activateSelect`) can be used to create a selection quickly, and then in Visual Mode `c` can turn to VisualMode's "caret mode".

If you want to enter caret mode directly, you may add such mappings:
```
map xxx enterVisualMode mode="caret"
map yyy LinkHints.activateSelect caret
```
